woocommerce/woocommerce-android
# issue/3906-save-reordered-attribute-terms

Fixes #3906 - re-ordered attribute terms weren't be retained previously, so the order was lost as soon as you left the screen. This PR corrects this, and also adds a change from anirudh bhat that fixes the animation during re-ordering.

To test:

- Visit the terms list for a local product attribute
- Re-order the terms
- Ensure the re-order is retained after leaving the screen

Note: this should be tested using _local_ attribute terms. We support re-ordering global attribute terms, but we shouldn't. I opened #3923 to address this.

Update release notes:
